config.env

```diff
@@ -0,0 +1,15 @@
+# GENERAL CONFIGURATION VARIABLES
+
+INGRESS_HOST=0.0.0.0
+INGRESS_PORT=80
+INGRESS_PATH=/
+EGRESS_URL=
+EXECUTE_SINGLE_COMMAND=no
+SINGLE_COMMAND=
+MELITA_API_URL=
+AUTHENTICATION_API_KEY=
+ERROR_URL=
```

Comment:
Could you provide some examples for these settings? If anyone wants to test, we are not sure what the settings should look like. Of course you can hide secrets or autogenerated values with some placeholders. For instance, instead of providing `EGRESS_URL=https://hookb.in/8Pd3VBykn0ipLGKKrnWd` we can add here `EGRESS_URL=https://hookb.in/example`

Reply:
added https://testmp.free.beeceptor.com